code/modules/cargo/packs.dm

```diff
@@ -3108,9 +3108,26 @@
 /datum/supply_pack/costumes_toys/randomised/plush/fill(obj/structure/closet/crate/C)
 	var/plush
 	for(var/i in 1 to num_contained)
-		plush = pick(subtypesof(/obj/item/toy/plush))
+		plush = pick(subtypesof(/obj/item/toy/plush) - /obj/item/toy/plush/carpplushie/dehy_carp)
 		new plush(C)
 
+/datum/supply_pack/costumes_toys/randomised/plush_no_moths
+	name = "Plushie Crate Without Moth Plushies"
+	desc = "A crate filled with 5 plushies without all those pesky moth plushies! Might contain dangerous plushies."
+	contraband = TRUE
+	cost = 1500
+	max_supply = 5
+	num_contained = 5
+	contains = list()
+	crate_type = /obj/structure/closet/crate/wooden
+	crate_name = "plushie crate"
+
+/datum/supply_pack/costumes_toys/randomised/plush_no_moths/fill(obj/structure/closet/crate/C)
+	var/plush_nomoth
+	for(var/i in 1 to num_contained)
+		plush_nomoth = pick(subtypesof(/obj/item/toy/plush) - typesof(/obj/item/toy/plush/moth))
+		new plush_nomoth(C)
```

Comment:
These lines actually mean:
```dm
	var/plush_nomoth
	for(var/i in 1 to num_contained)
		var/_temporary_list = subtypesof(/obj/item/toy/plush) - typesof(/obj/item/toy/plush/moth)
		plush_nomoth = pick(_temporary_list )
		new plush_nomoth(C)
```
You can see `var/_temporary_list = subtypesof(/obj/item/toy/plush) - typesof(/obj/item/toy/plush/moth)` is in the loop. This means you're creating a list of items every time this loop is continued.
but calculation can be done for the first time.
```dm
	var/plush_nomoth
	var/_temporary_list = subtypesof(/obj/item/toy/plush) - typesof(/obj/item/toy/plush/moth)
	for(var/i in 1 to num_contained)
		plush_nomoth = pick(_temporary_list )
		new plush_nomoth(C)
```
Now you don't calculate the list again and again.

Reply:
Alright a new commit should be up.

<details>
<summary>And here is me spawning one of each crate with the cargo spawn verb:</summary>

![dreamseeker_N2psD7B3rX](https://github.com/BeeStation/BeeStation-Hornet/assets/105242196/d02196ac-31ad-4c5b-a590-7c02e593bd00)
</details>
The runtime happened a few seconds after loading and I don't believe it's related to this PR, do correct me if I'm wrong!